### PR TITLE
Stock span #709

### PR DESCRIPTION
**Pull request by Shraddha13Jain**
Successfully implemented Stock Span problem using stack in python.
issue [https://github.com/Algo-Phantoms/Algo-Tree/issues/709]

### CODE REVIEW

**Review by raksha009 — COMMENTED** [review]
Link yr code to the appropriate folder

**Comment by Shraddha13Jain** [comment]
> Link yr code to the appropriate folder

done.

File: Stack/readme.md [hunk 1]
```diff
@@ -26,5 +26,7 @@ element in a full stack is called **overflow**.
 -   Evaluation of postfix expression ----> [C++](/Code/C++/Postfixexpression.cpp)
 -   Stack Class ----> [C++](/Code/C++/stack_class.cpp)
 -   Balanced Bracket Problem ----> [C++](/Code/C++/balanced_bracket.cpp)
+-   Stock Span Problem ----> [C++](/Code/C++/Stock_Span_Problem.cpp) | [Python](/Code/Python/StockSpan.py)
```
Comment (plazzy99) [inline Stack/readme.md:29]: I can see duplicate stock span problem please correct

**Review by plazzy99 — CHANGES_REQUESTED** [review]

File: Stack/readme.md (hunk 1 above)
Comment (Shraddha13Jain) [reply Stack/readme.md:29]: @plazzy99 I have corrected it..

**Comment by Shraddha13Jain** [comment]
@plazzy99  @rudrakshi99 please u may review it.. done the changes suggested

**Review by plazzy99 — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**